### DISCUSSION TITLE
왜 잔디가 안심어지는가(두둥탁)

### DISCUSSION THREAD

**Discussion by mtak0235** [mtak0235/helloPython#22] [Q&A]
💡문제 : 1일 1커밋을 했는데 왜 잔디가 안 심어질까요?
💡 원인: [ Default 브런치로 설정한 브런치(보통 main또는 master)에 다른 브런치의 커밋 내용을 풀리퀘스트 merge 해야지 커밋에 카운터가 됩니다.](https://okky.kr/article/930443)
> 그렇다면 잔디를 어떻게 하면 보이게 할 수 있을까?(두둥탁)

**Comment by DBrider3** [comment]
탁두둥
